### DISCUSSION TITLE
An Addition to Auto Hiding Block Palette Addon

### DISCUSSION THREAD

**Discussion by FlamedDogo99** [ScratchAddons/ScratchAddons#6408] [Ideas] [answered]
I propose an additional setting be added to the auto hiding block palette addon that allows the user to select the block palette category by hovering over the desired section. Just a small quality of life suggestion, not very important.

**Comment by WorldLanguages** [comment, marked as answer]
I guess this could be useful for the "category hover" mode, but the user would lose a way to open the palette "unchanged" without having to remember which category was previously open, we could probably try to make that experience better somehow (maybe we should add something else that when hovered shows the palette without changing the category)

In theory, this could also be its own separate addon which lets you hover categories to select them, even if the block palette is already open, but I'm not sure if that makes sense.
